hydra-node/src/Hydra/Chain/Direct.hs

```diff
@@ -347,6 +347,11 @@ txSubmissionClient tracer queue =
               -- possible because of missing data constructors from cardano-api
               let postTxError = FailedToPostTx{failureReason = show err}
               traceWith tracer PostingFailed{tx, postTxError}
+              -- NOTE: Delay callback in case our transaction got invalidated
+              -- because of a transaction seen in a block. This gives the
+              -- observing side of the chain layer time to process the
+              -- transaction and business logic might even ignore this error.
+              threadDelay 1
```

Comment:
This waiting seems a bit arbitrary and somewhat confusing. Would it really be enough to have an impact on slower real life networks?